platformcomponents/android/checkbox.json

```diff
@@ -0,0 +1,27 @@
+{
+  "checkbox": {
+    "figma": "https://www.figma.com/file/z6oi9Ku3X7rM1Wj2451B9u/Components---Android?node-id=591%3A93",
+    "checked":{
+      "#normal":{
+        "background": "@theme-control-active-normal",
+        "text": "@theme-text-primary-normal",
+        "icon": "@theme-text-primary-inverted"
+      },
+      "#disabled":{
+        "background": "@theme-control-active-disabled",
+        "text": "@theme-text-primary-disabled",
+        "icon": "@theme-text-primary-inverted"
+      }
+    },
+    "unchecked":{
+      "#normal":{
+        "background":"@theme-page-control-normal",
```

Comment:
This should still be a border rather than background:
![image](https://user-images.githubusercontent.com/56216033/131538760-f53129af-1554-4eab-aa94-ecf301675ba7.png)
Unchecked enabled is the only one with a border but no background; the rest have background but no border